### DISCUSSION TITLE
Dúvidas no Split com PIX

### DISCUSSION THREAD

**Discussion by dncosta** [bacen/pix-api#243] [Ideas]
Como viabilizar split com PIX?

Dado que nas APIs PIX ainda não é possível incluir dados de  um split de pagamento, qual seria a forma mais adequada de gerar um QRCode PIX numa API proprietária? Vejo três cenários, gostaria de saber se eles possuem limitações do Bacen.

1 - Gerar o QRCode para um dos recebedores, mas no momento da liquidação dividir o pagamento.
Esse modelo talvez não esteja compliance com as normais do PIX e do Bacen.

2 - Gerar o QRcode para um dos recebedores, liquidar 100% do valor pra esse recebedor de em seguida transferir o valor do segundo recebedor.
Esse modelo gera dificuldades para os PSPs, dado que normalmente trabalha-se dividindo os recebíveis já na agenda ou na liquidação.

3 - Gerar o QRcode com uma chave do PSP e um ID que identifique as caracteristicas do split para efetivar a liquidação corretamente após a confirmação.
A dificuldade aqui é de garantir a compreensão do comprador, quando perceber que o QRCode criado não está endereçado para o vendedor, mas sim para um intermediário.

**Comment by rubenskuhl** [comment]
Já há muitos pagamentos feitos hoje para intermediários (Mercado Pago, Juno, Adyen, PagSeguro etc.) e as pessoas físicas não implicam com isso (já empresa com área financeira não gosta). Me parece que o ideal por enquanto é o destinatário ser uma conta do intermediário, e ele fazer a divisão.

**Comment by dncosta** [comment]
@rubenskuhl , eu também tenho a impressão de que a existência de um intermediário é necessária. Seguindo a abordagem 2 ou 3.

Porém, não encontrei nenhum trecho normativo que invalide a possibilidade de liquidar uma transação do PIX em 2 recebedores como proposto no cenário 1.

Somado isso com o trecho da [Resolução BCB n° 30 de 29/10/2020](https://www.bcb.gov.br/estabilidadefinanceira/exibenormativo?tipo=Resolu%C3%A7%C3%A3o%20BCB&numero=30) mostrada abaixo talvez nos dê essa margem.

```
Subseção III
Da API Pix

Art. 15-B.  Os participantes do Pix que disponibilizem funcionalidades aos usuários finais relacionadas à API devem realizá-lo por meio da API Pix.
Parágrafo único.  É facultada a oferta, pelos participantes, em APIs próprias, de funcionalidades acessórias ou complementares àquelas disponibilizadas na API Pix.” (NR)
```

Em teoria, numa API proprietária eu poderia permitir que fossem informados 2 ou mais recebedores e permitir que a liquidação acontecesse pra todos desde que garantida a liquidação integral.

Faz sentido?

**Comment by rubenskuhl** [comment]
Faz. Enquanto isso não existir na API padronizada, pode ser oferecido em API proprietária. Porém, já há discussões de como implementar isso na API padronizada, vide https://github.com/bacen/pix-api/issues/229 . Aí a questão é de time-to-market e de migração posterior para o padrão.

**Comment by dncosta** [comment]
Boa.
